packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Navigation.tsx

```diff
@@ -81,14 +98,14 @@ const NavigationList = (props: NavigationProps) => {
   let children: JSX.Element[] = [];
   if (props.fields.Children && props.fields.Children.length) {
     children = props.fields.Children.map((element: Fields, index: number) => (
-      <NavigationList key={`${index}${element.Id}`} fields={element} />
+      <NavigationList key={`${index}${element.Id}`} fields={element} handleClick={props.handleClick} />
     ));
   }
 
   return (
     <li className={props.fields.Styles.join(' ')} key={props.fields.Id}>
       <div className="navigation-title">
-        <Link field={getLinkField(props)} title={title}>
+        <Link field={getLinkField(props)} title={title} onClick={() => props.handleClick(false)}>
```

Comment:
use `onClick={props.handleClick}` `handleClick` doesnt accept parameters

Reply:
Parameters are used in here